### DISCUSSION TITLE
Budibase attachment column

### DISCUSSION THREAD

**Discussion by dado-ca** [Budibase/budibase#14412] [Help]
Hi, I have an issue where in a single attachment column, the image preview stops working after 10 - 15 min. On the frontend also, I successfully upload the image, and I am able to show it on the frontend, after 10-15 min it stops showing up. When I click on the image url, in the data section however it downloads the image.

How can I solve this issue?

![budibase attachment preview](https://github.com/user-attachments/assets/2055f141-2c1e-422f-9888-bb1c2f157a37)

**Comment by dado-ca** [comment]
Tried to solved it this way https://github.com/Budibase/budibase/discussions/7777. It found one conflict, I resolved it. But images are still not showing

**Comment by mikesealey** [comment]
Hey @dado-ca 

I'm sorry to see you've having trouble. This looks a little bit like a bug, but I'd like to know a bit more information if you can help me out. 

Firstly, are you cloud-hosting or self-hosted? 
What version are you running?
Does refreshing the page fix the preview?
Does making a change to something else in the row fix the preview?
If you create a view, does the preview work in the view? (either for a prolonged duration, or failing after 10-15 minutes as described)

Are you able to find the image in the cell through the browser's developer tools? Are there any clues in the html `<img>` tags (or nearby)?
![image](https://github.com/user-attachments/assets/2370925b-e1c0-450d-b414-30be9523cd4d)

**Reply by dado-ca** [reply]
Hi there, well, I think it will be solved. 

I'am self hosting budibase in docker, with persistent volume. 

When viewing the attachment in Developer tools I,am getting error 403:

I tried changing the Access level to public, but nothing happens.

Refreshing the page, and viewing it from incognito mode does not work eighter.

Creating a view, same result.
 



![Budibase attachment view error](https://github.com/user-attachments/assets/896d5919-1063-45cd-93c8-9fa73d30bcfe)
